sdks/csharp/twin_operations_other.cs

```diff
@@ -1,74 +1,99 @@
-// ------------------ CREATE TWIN: NO HELPER ---------------------
-// <CreateTwin_noHelper>
-// Define a custom model type for the twin to be created
-class CustomDigitalTwin
+using Azure;
+using Azure.DigitalTwins.Core;
+using System;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace DigitalTwins_Samples
 {
-    [JsonPropertyName(DigitalTwinsJsonPropertyNames.DigitalTwinId)]
-    public string Id { get; set; }
+    // ------------------ CREATE TWIN: NO HELPER ---------------------
+    // <CreateTwin_noHelper>
+    // Define a custom model type for the twin to be created
 
-    [JsonPropertyName(DigitalTwinsJsonPropertyNames.DigitalTwinETag)]
-    public string ETag { get; set; }
+    internal class CustomDigitalTwin
```

Comment:
@cippero  Did we ever get clarification on whether internal or public is appropriate here? I see other classes in this file are public, so wondering about the difference and confirming why these 2 are internal instead.